README.md

```diff
@@ -58,6 +58,15 @@ you can disable it using the `--no-haproxy.ssl-verify` flag:
 haproxy_exporter --no-haproxy.ssl-verify --haproxy.scrape-uri="https://haproxy.example.com/haproxy?stats;csv"
 ```
 
+If scraping a remote HAProxy must be done via an HTTP proxy, you can enable reading of the
+standard `$HTTP_PROXY` / `$HTTPS_PROXY` / `$NO_PROXY` environment variables by using the
```

Comment:
```suggestion
standard [`$http_proxy` / `$https_proxy` / `$no_proxy` environment variables](https://pkg.go.dev/net/http#ProxyFromEnvironment) by using the
```

Reply:
+1 for the documentation link, and no opinion about uppercase/lowercase variables names (so I will go with your preference).